CRM/Contribute/Form/Contribution/Confirm.php

```diff
@@ -2055,10 +2055,6 @@ public static function submit($params) {
     $form->controller = new CRM_Contribute_Controller_Contribution();
     $params['invoiceID'] = md5(uniqid(rand(), TRUE));
 
-    // We want to move away from passing in amount as it is calculated by the actually-submitted params.
```

Comment:
This function is only used by ContributionPage.submit api which is likely only used but the tess